resources/web/docs_js/index.js

```diff
@@ -267,5 +267,22 @@ $(function() {
     document.head.appendChild(hotcss);
   }
 
+  // For the private docs repositories, the edit button is hidden
+  // unless there is an '?edit' in the query string or hash.
+  if (utils.deparam(window.location.search).edit !== undefined
+      || window.location.hash.indexOf('?edit') > -1) {
```

Comment:
Do we need to do this for the anchor? That feels a little funky.

Reply:
I think we should handle both for a simple (yet weird) reason:  The workflow for this feature is to navigate to a page, see that something is broken, click the URL, add `?edit` at the end, hit enter, and click the edit button (not ideal, i know).  The problem with the above is that if the URL ends with an anchor, adding `?edit` to it wouldn't do anything, so the user would have to find the beginning of the anchor and edit there. If we can support this weird case out of the box, user won't have to worry about it.

Reply:
This ok with you @nik9000 ? I don't see how `?edit` in the hash part would cause problems with our own use of anchors.